content.md

```diff
@@ -2,18 +2,24 @@ A collection of tools and resources to enhance your gameplay!
 
 -# Is something missing? Contribute to this list on [GitHub](https://github.com/osucord/resources).
 ---
+### Self-improvement
+- [khz's streaming guide](https://docs.google.com/document/d/1xbNwH_vN4O3azBYwua-Z_3GOAI2d-kV_EeE96aiRLBU/edit) - A guide on how to get better at streaming
+- [Shikke's Anti-Mindblock](https://shikkesora.com/downloads) - A tool to quickly flip your monitor upside to help with mind-block
+- [osu!mania Pattern Repository](https://imgur.com/a/replacement-glossary-osu-mania-pattern-repository-qEwP37W) - A glossary of common osu!mania patterns and terms
+---
```

Comment:
this should not be at the top

Reply:
I think most players start with the question "how do I get better at the game", then worry about peripherals. That's why I added it to the top. Where would you suggest moving it?

Reply:
considering these are just random, subjective, guides by random players, I don't think it should be near the top at all.. maybe under training tools? but honestly the whole ordering needs work

Reply:
the mania patterns is also not related to self improvement I think but moreso mapping/skill knowledge? should stay in misc

Reply:
Yeah, I thought it's a really cool resource, but I couldn't find a better place for it. I also don't like the "Misc" category - it feels kind of dismissive to put something there, as if saying "you're not important enough to be properly categorized".

Reply:
maybe just put it under mapping resources actually (which fwiw should be renamed to just "Mapping". It's rather redundant with the current title)

Reply:
Its not a lack of importance its a lack of quantity, if you have other similar resources feel free to split it off under its own header.

Reply:
It's not really a mapping resource, it's a glossary for how both players and mappers communicate about mania. Maybe we can find more of these glossaries for other modes and combine them into their own category

Reply:
then put it in misc, it's not personal improvement at the very least

Reply:
Alright, moved